CMake config setup bug

When @gaede fixed the cmakeconfig path here https://github.com/iLCSoft/iLCInstall/commit/e69a7d154ef2392ddc94b5e7b395174858698ab5#diff-43c7f15e4148cc1085c2974bcaf23412R90
```py
self.cmakeconfig = self.installPath + "/lib/Geant4-" + v  + "; " + self.installPath + "/lib64/Geant4-" + v 
```
a semicolon was introduced, but cmakeconfig is used in ilcsoft.py to set two things: 1. the bash script and 2. the cmake env https://github.com/iLCSoft/iLCInstall/blob/bacc1a298b427070d8f2a7923d5b295e26113d41/ilcsoft/ilcsoft.py#L282-L283 . With the intro semicolon it breaks the propper cmake setup.

The easiest fix is that you check in geant4.py which system it is and then set lib or lib64, but you cannot set both.

Not there is a problem in MarlinKinFit
```
-- Check for MarlinUtil_MARLINUTIL_LIBRARY: /cvmfs/clicdp.cern.ch/iLCSoft/builds/nightly/x86_64-slc6-gcc62-opt/MarlinUtil/HEAD/lib/libMarlinUtil.so -- ok
-- Check for MarlinUtil_MARLINUTILANN_LIBRARY: /cvmfs/clicdp.cern.ch/iLCSoft/builds/nightly/x86_64-slc6-gcc62-opt/MarlinUtil/HEAD/lib/libMarlinUtilAnn.so -- ok
-- Found MarlinUtil: /cvmfs/clicdp.cern.ch/iLCSoft/builds/nightly/x86_64-slc6-gcc62-opt/MarlinUtil/HEAD (Required is at least version "1.4") 
CMake Error at CMakeLists.txt:48 (FIND_PACKAGE):
  By not providing "FindMarlinKinfit.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "MarlinKinfit", but CMake did not find one.

  Could not find a package configuration file provided by "MarlinKinfit"
  (requested version 0.0) with any of the following names:

    MarlinKinfitConfig.cmake
    marlinkinfit-config.cmake

  Add the installation prefix of "MarlinKinfit" to CMAKE_PREFIX_PATH or set
  "MarlinKinfit_DIR" to a directory containing one of the above files.  If
  "MarlinKinfit" provides a separate development package or SDK, be sure it
  has been installed.
```

https://gitlab.cern.ch/CLICdp/SoftwareConfigurations/iLCSoft/-/jobs/6487208

--- a/builds/release-ilcsoft.cfg
+++ b/builds/release-ilcsoft.cfg
@@ -121,6 +121,7 @@ ilcsoft.install( MarlinPKG( "LCTuple", LCTuple_version ))
 ilcsoft.module("LCTuple").addDependency( [ 'LCIO', 'Marlin', 'ROOT'] )
 
 ilcsoft.install( MarlinPKG( "MarlinKinfit", MarlinKinfit_version ))
+ilcsoft.module("MarlinKinfit").hasCMakeFindSupport=True
 ilcsoft.module("MarlinKinfit").addDependency( [ 'LCIO', 'GEAR', 'GSL', 'Marlin'] )
 
 ilcsoft.install( MarlinTrk( MarlinTrk_version ))
@@ -145,6 +146,8 @@ ilcsoft.install( MarlinPKG( "Physsim", Physsim_version ))
 ilcsoft.module("Physsim").addDependency( [ 'LCIO', 'ROOT', 'Marlin' ] )
 
 ilcsoft.install( MarlinPKG( "FCalClusterer", FCalClusterer_version ))
+# Special case: BeamCalRecoConfig.cmake is generated in FCalClusterer
+ilcsoft.module("FCalClusterer").hasCMakeFindSupport=True
 ilcsoft.module("FCalClusterer").download.supportedTypes = [ "GitHub" ]
 ilcsoft.module("FCalClusterer").download.gituser = 'FCALSW'
 ilcsoft.module("FCalClusterer").download.gitrepo = 'FCalClusterer'
@@ -156,6 +159,7 @@ ilcsoft.install( MarlinPKG( "ForwardTracking", ForwardTracking_version ))
 ilcsoft.module("ForwardTracking").addDependency( [ 'LCIO', 'GEAR', 'ROOT', 'GSL', 'Marlin', 'MarlinUtil', 'MarlinTrk'] )
 
 ilcsoft.install( MarlinPKG( "Clupatra", Clupatra_version ))
+ilcsoft.module("Clupatra").hasCMakeFindSupport=True
 ilcsoft.module("Clupatra").addDependency( [ 'LCIO', 'ROOT', 'RAIDA', 'Marlin', 'MarlinUtil', 'KalTest', 'MarlinTrk' ] )
 
 ilcsoft.install( PathFinder( PathFinder_version ))

